[BUG] Mypy check error in newest pandas-stubs version

### Observed behavior

I run the [ci on my cloned repo](https://github.com/shenxiangzhuang/crowd-kit/actions/runs/9251925080) with the error in python >= 3.9 as fellows:
```
Run mypy crowdkit tests
crowdkit/aggregation/classification/glad.py:160: error: Unexpected keyword argument "copy" for "merge"  [call-arg]
/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/pandas-stubs/core/reshape/merge.pyi:22: note: "merge" defined here
Found 1 error in 1 file (checked 79 source files)
Error: Process completed with exit code 1.
```

I just lookup the changelog of pandas-stubs package, and I do find the delete operation of the `coyp` parameter: https://github.com/pandas-dev/pandas-stubs/pull/904/files#diff-8867b41003793df20a65f8c9bf8b4085caf372f603165a8fc0d9dad51ca37441

According to [the newest document of pandas](https://pandas.pydata.org/docs/reference/api/pandas.merge.html), maybe we can just remove the usage of `copy` here. I will create a pr for this later and feel free the give your suggestions about this!

### Expected behavior

_No response_

### Python Version

3.9

### Crowd-Kit Version

1.3.0.post0

### Other Packages Versions

_No response_

### Example code

```Python
Just trigger the ci, the mypy error will be raised
```


### Relevant log output

_No response_

--- a/crowdkit/aggregation/classification/glad.py
+++ b/crowdkit/aggregation/classification/glad.py
@@ -157,9 +157,7 @@ class GLAD(BaseClassificationAggregator):
         probas = probas.groupby(["task"]).transform(self._softmax)
         # put posterior in data['posterior']
         probas.name = "posterior"
-        data = pd.merge(
-            data.drop("posterior", axis=1), probas, on=["task", "variable"], copy=False
-        )
+        data = pd.merge(data.drop("posterior", axis=1), probas, on=["task", "variable"])
 
         self.probas_ = probas.unstack()
         return data

--- a/crowdkit/aggregation/classification/glad.py
+++ b/crowdkit/aggregation/classification/glad.py
@@ -158,7 +158,7 @@ class GLAD(BaseClassificationAggregator):
         # put posterior in data['posterior']
         probas.name = "posterior"
         data = pd.merge(
-            data.drop("posterior", axis=1), probas, on=["task", "variable"], copy=False  # type: ignore[call-arg]
+            data.drop("posterior", axis=1), probas, on=["task", "variable"], copy=False  # type: ignore[call-arg, unused-ignore]
         )
 
         self.probas_ = probas.unstack()

--- a/crowdkit/aggregation/classification/glad.py
+++ b/crowdkit/aggregation/classification/glad.py
@@ -158,7 +158,7 @@ class GLAD(BaseClassificationAggregator):
         # put posterior in data['posterior']
         probas.name = "posterior"
         data = pd.merge(
-            data.drop("posterior", axis=1), probas, on=["task", "variable"], copy=False  # type: ignore[call-arg, unused-ignore]
+            data.drop("posterior", axis=1), probas, on=["task", "variable"], copy=False  # type: ignore[call-arg,unused-ignore]
         )
 
         self.probas_ = probas.unstack()

--- a/crowdkit/aggregation/classification/glad.py
+++ b/crowdkit/aggregation/classification/glad.py
@@ -157,7 +157,9 @@ class GLAD(BaseClassificationAggregator):
         probas = probas.groupby(["task"]).transform(self._softmax)
         # put posterior in data['posterior']
         probas.name = "posterior"
-        data = pd.merge(data.drop("posterior", axis=1), probas, on=["task", "variable"])
+        data = pd.merge(
+            data.drop("posterior", axis=1), probas, on=["task", "variable"], copy=False  # type: ignore[call-arg]
+        )
 
         self.probas_ = probas.unstack()
         return data

